.travis.yml

```diff
@@ -1,5 +1,5 @@
 language: python
-dist: xenial
+dist: bionic
```

Comment:
This was necessary because `openjdk-11-jre-headless` is not available on xenial. https://packages.ubuntu.com/search?keywords=openjdk-11-jre-headless&searchon=names